runners/direct-java/src/main/java/org/apache/beam/runners/direct/EvaluationContext.java

```diff
@@ -159,7 +159,8 @@ public CommittedResult handleResult(
         completedBundle == null
             ? null
             : completedBundle.withElements((Iterable) result.getUnprocessedElements()),
-        committedBundles);
+        committedBundles,
+        result.producedOutput());
```

Comment:
In isolation there is actually a readability concern that `producedOutput` can read as a noun.